lib/evss/service.rb

```diff
@@ -65,25 +65,16 @@ def handle_error(error)
 
       case error
       when Faraday::ParsingError
-        raise_backend_exception('EVSS502', self.class)
+        raise_backend_exception('EVSS502', "EVSS::#{self.class}")
```

Comment:
The `EVSS::` prefix isn't necessary.  This results in `EVSS::EVSS::Service` being sent.

Reply:
Fixing this 😄